### DISCUSSION TITLE
how do you animate the way accordion opens

### DISCUSSION THREAD

**Discussion by aquibbaig** [radix-ui/primitives#1276] [Help]
Hey guys, first of all kudos to the team for creating such an awesome tool 🥳 

I wanted to add an animation property that would customise the default animation when an accordion content shows and delay it a little bit with an animation such as "easeIn", but failed to do so. I have tried with framer and react-spring but the animation gets applied to the content inside it. Is there a way you can customise the animation for the way the accordion opens?

**Comment by benoitgrelard** [comment]
Hi @aquibbaig,

There's an example in our docs which is animated and has custom easing like you mention: https://www.radix-ui.com/docs/primitives/components/accordion
Here's the sandbox for it: https://codesandbox.io/s/ln1z3v?module=App.js
